cli/src/account.rs

```diff
@@ -1,34 +1,82 @@
 // Copyright 2020-2022 IOTA Stiftung
 // SPDX-License-Identifier: Apache-2.0
 
+use std::borrow::Cow;
+
 use clap::Parser;
 use colored::Colorize;
-use dialoguer::Input;
 use iota_sdk::wallet::{Account, Wallet};
+use rustyline::{
+    error::ReadlineError, highlight::Highlighter, hint::HistoryHinter, history::MemHistory, Completer, Config, Editor,
+    Helper, Hinter, Validator,
+};
 
 use crate::{
-    account_completion::AccountCompletion,
-    account_history::AccountHistory,
-    command::account::{
-        addresses_command, balance_command, burn_native_token_command, burn_nft_command, claim_command,
-        claimable_outputs_command, consolidate_command, create_alias_outputs_command, create_native_token_command,
-        decrease_voting_power_command, destroy_alias_command, destroy_foundry_command, faucet_command,
-        increase_voting_power_command, melt_native_token_command, mint_native_token, mint_nft_command,
-        new_address_command, node_info_command, output_command, outputs_command, participation_overview_command,
-        send_command, send_native_token_command, send_nft_command, stop_participating_command, sync_command,
-        transaction_command, transactions_command, unspent_outputs_command, vote_command, voting_output_command,
-        voting_power_command, AccountCli, AccountCommand,
+    command::{
+        account::{
+            addresses_command, balance_command, burn_native_token_command, burn_nft_command, claim_command,
+            claimable_outputs_command, consolidate_command, create_alias_outputs_command, create_native_token_command,
+            decrease_voting_power_command, destroy_alias_command, destroy_foundry_command, faucet_command,
+            increase_voting_power_command, melt_native_token_command, mint_native_token, mint_nft_command,
+            new_address_command, node_info_command, output_command, outputs_command, participation_overview_command,
+            send_command, send_native_token_command, send_nft_command, stop_participating_command, sync_command,
+            transaction_command, transactions_command, unspent_outputs_command, vote_command, voting_output_command,
+            voting_power_command, AccountCli, AccountCommand,
+        },
+        account_completion::AccountCompleter,
     },
     error::Error,
     helper::bytes_from_hex_or_file,
     println_log_error,
 };
 
+#[derive(Helper, Completer, Hinter, Validator)]
```

Comment:
This should probably move to the `account_completion` module

Reply:
Done